.github/workflows/e2e_api_tests.yml

```
@@ -236,3 +245,22 @@
                       tests/pw/playwright-report
                   if-no-files-found: ignore
                   retention-days: 30
+
+            - name: Publish HTML report to GH Pages
+              if: always() && steps.generate-html-report.outcome == 'success'
```

actionlint (1.7.4)

property "generate-html-report" is not defined in object type {clone-dokan-lite: {conclusion: string; outcome: string; outputs: {commit: string; ref: string}}; debug-log: {conclusion: string; outcome: string; outputs: {string => string}}; e2e-test: {conclusion: string; outcome: string; outputs: {string => string}}; playwright-cache: {conclusion: string; outcome: string; outputs: {cache-hit: string}}; playwright-version: {conclusion: string; outcome: string; outputs: {string => string}}; pw-install: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)